### PR TITLE
Chicken Bar

### PR DESCRIPTION
**Pull request by chids**

### CODE REVIEW

**Comment by hnrc** [comment]
"This branch has conflicts that must be resolved" 😿

**Comment by chids** [comment]
⚡

**Comment by hnrc** [comment]
Wiiii 👏 🙇 🍻

File: index.html [hunk 1]
```diff
@@ -59,6 +59,7 @@
     <div id="babajan" data-url="https://untappd.com/v/babajan/162502" class="venue"><h1>Babajan</h1></div>
     <div id="bishops-arms-folkungagatan" data-url="https://untappd.com/v/bishops-arms/2578" class="venue"><h1>Bishops Arms Folkungagatan</h1></div>
     <div id="the-queens-head" data-url="https://untappd.com/v/queens-head/19721" class="venue"><h1>The Queens Head</h1></div>
+    <div id="chicken-bar" data-url="https://untappd.com/v/chicken-bar-stockholm/3707630" class="venue"><h1>Chicken Bar</h1></div>
```
Comment (hnrc) [inline index.html:62]: This sucks

Comment (hnrc) [reply index.html:62]: ok nvm

**Review by hnrc — COMMENTED** [review]
👍

**Pull request merged**